src/plone/restapi/services/contact/post.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from plone.restapi.services import Service
+from zope.component import getMultiAdapter
+from plone.restapi.deserializer import json_body
+
+
+class ContactPost(Service):
+
+    def reply(self):
+        data = json_body(self.request)
+
+        sender_from_address = data.get('from', None)
+        message = data.get('message', None)
+        sender_fullname = data.get('name', '')
+        subject = data.get('subject', '')
+
+        if not sender_from_address or not message:
+            self.request.response.setStatus(400)
+            return dict(error=dict(
+                type='BadRequest',
+                message='Missing from or message parameters'))
+
+        ctrlOverview = getMultiAdapter((self.context, self.request),
```

Comment:
@sneridagh we should use pep8 naming conventions here and not use abbreviations in variable names. "ctrlOverview" -> overview_controlpanel.

Reply:
Done.